docs/how-to/upgrade.md

```diff
@@ -1,16 +1,26 @@
 # Upgrade
 
-!!! caution alert alert-warning "Important"
-    Upgrade the PMM Server before you upgrade the PMM Client.
-    Ensure that the PMM Server version is higher than or equal to the PMM Client version. Otherwise, there might be configuration issues, thus leading to failure in the client-server communication as PMM Server might not be able to identify all the parameters in the configuration.
+## Plan the upgrade order
+
+### Upgrade PMM Server first
+
+Make sure to upgrade the PMM Server before you upgrade the PMM Client.
+    
+Ensure that the PMM Server version is higher than or equal to the PMM Client version. Otherwise, there might be configuration issues, thus leading to failure in the client-server communication as PMM Server might not be able to identify all the parameters in the configuration.
+
+For example, for a PMM Server version 2.25.0, the PMM Client version should be 2.25.0 or 2.24.0. If the PMM Client version is 2.26.0, PMM might not work as expected.
+
+### Staged upgrade approach
+
+When upgrading PMM from older versions (2.30.0 and below), we recommend following a staged approach: first, upgrade to version 2.32.0, and then proceed to the latest version. 
```

Comment:
```suggestion
When upgrading PMM from older versions (2.30.0 and below), we recommend following a staged approach: first, upgrade to version 2.33.0, and then proceed to the latest version. 
```
Below we are not recommending to upgrade to 2.32, so let's ask to upgrade to 2.33 instead :)